libc/utils/gpu/server/rpc_server.cpp

```diff
@@ -21,6 +21,11 @@
 
 using namespace LIBC_NAMESPACE;
 
+// Workaround for missing __has_builtin in < GCC 10.
+#ifndef __has_builtin
+#define __has_builtin 0
+#endif
+
```

Comment:
Needs to be above the `#include`s.

Reply:
I think it also need to be `#define __has_builtin(x) 0` otherwise we'll get a failure during preprocessing.  The macro expansion sites expect a function like macro.

Reply:
Done.

Reply:
Sorry, this MUST be fixed.
https://godbolt.org/z/rzhe1Ee7f

Reply:
Duh, thanks, needs a macro argument.

Reply:
Should be fixed with latest commit.